internal/lib/lib.go

```diff
@@ -1337,11 +1337,19 @@ func PassthroughShardSize() uint32 {
 	}
 	return 1
 }
-
+func GetAKOID_Prefix() string {
```

Comment:
please consider removing the `_` from the function name.

Reply:
Addressed